### DISCUSSION TITLE
DNS error

### DISCUSSION THREAD

**Discussion by branzllle** [vercel/community#2964] [Help] [answered]
### Summary

I recently changed my hosting from hostinger to github. My site can accessed via Kproxy but unable to access via my phone and computer.

[www.biscubeuae.com
](https://biscube.branwith.com)
But a subdomain I connected working very fine and quickly. 
[biscube.branwith.com
](https://biscube.branwith.com)
kproxy
[Kproxy link](https://server5.kproxy.com/servlet/redirect.srv/sruj/snkfqoirizt/sqrs/p2/)

### Example

https://www.biscubeuae.com/

### Steps to Reproduce

**My DNS entries**
![image](https://github.com/vercel/community/assets/87488089/198396d7-3b92-408d-ae5a-06de971f9ba1)

**Nameserver**
![image](https://github.com/vercel/community/assets/87488089/173e820f-87c5-4a50-a1b6-4a502ca348eb)

**Error**
![image](https://github.com/vercel/community/assets/87488089/2164a0b3-fca3-46ef-9d88-827e73362229)

**Comment by smaeda-ks** [comment, marked as answer]
@branwith are you still seeing the issue? It might have been a DNS propagation issue.
